token/cli/src/main.rs

```diff
@@ -2346,27 +2352,39 @@ impl offline::ArgsConfig for SignOnlyNeedsFullMintSpec {
     fn sign_only_arg<'a, 'b>(&self, arg: Arg<'a, 'b>) -> Arg<'a, 'b> {
         arg.requires_all(&[MINT_ADDRESS_ARG.name, MINT_DECIMALS_ARG.name])
     }
+    fn signer_arg<'a, 'b>(&self, arg: Arg<'a, 'b>) -> Arg<'a, 'b> {
+        arg.requires_all(&[MINT_ADDRESS_ARG.name, MINT_DECIMALS_ARG.name])
```

Comment:
iirc, `requires_all()` discards existing, rather than appending.  so i think we need to explicitly redeclare `BLOCKHASH_ARG.name` here

Reply:
oh n/m i'm full of shit, as usual.

https://docs.rs/clap/2.34.0/src/clap/args/arg.rs.html#1784-1787

Reply:
Looks like it adds according to the impl: https://docs.rs/clap/2.34.0/src/clap/args/arg.rs.html#1783